### PR TITLE
Update configurable-product-feature-overview.md

### PR DESCRIPTION
**Pull request by jeremyfourna**
Fix issues in Grammarly and add shopping list and wish list sections. Also, remove the beta mention.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by lenadoc — DISMISSED** [review]

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
-last_updated: Nov 26, 2021
+last_updated: Nov 29, 2022
 template: concept-topic-template
 redirect_from:
   - /docs/scos/user/features/201903.0/configurable-product-feature-overview.html
   - /docs/scos/user/features/201907.0/configurable-product-feature-overview.html
 ---
 
-{% info_block errorBox "Beta version" %}
+The *Configurable Product* feature introduces a new type of product that customers—a configurable product, can customize.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:11]:
```suggestion
The *Configurable Product* feature introduces a new type of product, a configurable product, that customers can customize.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
 
-This is the Beta version of the feature and is therefore subject to changes.
-
-{% endinfo_block %}
-
-The *Configurable Product* feature introduces a new type of product that can be customized by customers—a configurable product.
-
-The feature enables you to sell complex products with modular designs or services. For example, if you sell clothes, you can allow your customers to define the material and color and add their name to the product. Or, if you are selling a service, you can allow them to select a preferred date and time of the service delivery.
+The feature lets you sell complex products with modular designs or services. For example, if you sell clothes, your customers can define the material and color and add their names to the product. Or, if you are selling a service, you can allow them to select a preferred date and time of the service delivery.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:13]:
```suggestion
The feature lets you sell complex products with modular designs or services. For example, if you sell clothes, your customers can define the material and color and add their names to the product. Or if you are selling a service, you can allow them to select a preferred date and time for the service delivery.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 3]
```diff
@@ -26,7 +20,7 @@ For example, if you are selling a workstation installation service, before purch
 
 ### Configuring a configurable product
 
-To configure a product, from the **Product Details** page, a customer opens a product configurator by clicking the **Configure** button. Then, they are redirected back to the **Product Details** page and can add the configured product to the wishlist or cart.
+To configure a product, a customer opens a product configurator from the **Product Details** page by clicking the **Configure** button. Then, they are redirected back to the **Product Details** page and can add the configured product to the wishlist or cart.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:23]:
```suggestion
To configure a product, a customer opens a product configurator from the *Product Details* page by clicking the **Configure** button. They are then redirected back to the *Product Details* page and can add the configured product to the wishlist or cart.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 4]
```diff
@@ -41,20 +35,18 @@ Configurable products are created in two steps:
 1. A Back Office user creates regular products, or a developer imports them. See [Creating an abstract product](/docs/scos/user/back-office-user-guides/{{page.version}}/catalog/products/manage-abstract-products-and-product-bundles/create-abstract-products-and-product-bundles.html) to learn how they create products in the Back Office or [File details: product_concrete.csv](/docs/scos/dev/data-import/{{page.version}}/data-import-categories/catalog-setup/products/file-details-product-concrete.csv.html) to learn about the file they import.
 2. A developer converts regular products into configurable products by importing configuration parameters. See [File details: product_concrete_pre_configuration.csv](/docs/scos/dev/data-import/{{page.version}}/data-import-categories/special-product-types/configurable-product-import-category/file-details-product-concrete-pre-configuration.csv.html) to learn about the file they import.
 
-
 ### Managing configurable products
 
-A Back Office user can add configurable products to pages, categories, and content items as regular products.
+As regular products, a Back Office user can add configurable products to pages, categories, and content items.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:40]:
```suggestion
A Back Office user can add configurable products as regular products to pages, categories, and content items.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 5]
```diff
@@ -41,20 +35,18 @@ Configurable products are created in two steps:
...
 
-In the product catalog, they can see which products are configurable and edit them as regular products. However, a Back Office user cannot change configuration parameters.
+They can see which products are configurable in the product catalog and edit them as regular products. However, a Back Office user cannot change configuration parameters.
 
 ![configurable-product-entry-in-the-back-office](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Configurable+Product/Configurable+Product+feature+overview/configurable-product-entry-in-the-back-office.png)
 
-In the orders, A Back Office can see which products are configurable ones. They can also see the configuration of each product, but they cannot change the selected parameters.
+In the orders, A Back Office can see which products are configurable. They can also see the configuration of each product, but they cannot change the selected parameters.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:46]:
```suggestion
In the orders, a Back Office user can see which products are configurable. They can also see the configuration of each product, but they cannot change the selected parameters.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 6]
```diff
@@ -63,12 +55,11 @@ You can create a product configurator as a part of your shop or integrate a thir
...
 
-
 ### How a Spryker shop interacts with a product configurator
 
 A Spryker shop interacts with product configurators using parameters. When a customer is redirected from a Spryker shop to the configurator page, the shop passes the customer and product parameters. When the customer is redirected back to the shop, the configurator passes the updated parameters back.
 
-The behavior of the configurator is based on the parameters passed by a shop. For example, a shop passes the `store_name` parameter. If a customer is redirected from a US store, the language of the configurator is English. Also, the shop passes the URL of the page the customer is redirected from. After they save the configuration, the configurator uses this URL to redirect them back to the same page.
+The behavior of the configurator is based on the parameters passed by a shop. For example, a shop passes the `store_name` parameter. If a customer is redirected from a US store, the language of the configurator is English. Also, the shop passes the URL of the page the customer is redirected from. After they save the configuration, the configurator uses this URL to redirect them to the same page.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:62]:
```suggestion
The behavior of the configurator is based on the parameters passed by a shop. For example, a shop passes the `store_name` parameter. If a customer is redirected from a US store, the language of the configurator is English. Also, the shop passes the URL of the page the customer is redirected from. After the customer saves the configuration, the configurator uses this URL to redirect them back to the same page.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 7]
```diff
@@ -89,26 +80,25 @@ Display parameter values are usually converted from configuration parameter valu
...
 
-If the configuration is complete, on entering the **Product details** page, a customer sees a message that the configuration is complete. By default, the message is followed by the first three descriptive attributes set in the configurator. Under the attributes, there are the **Show** and **Hide** buttons that allow expanding and collapsing of the remaining attributes, respectively. In case the configuration is complete, the customer can purchase the product without opening the configurator and selecting parameters.
+If the configuration is complete, on entering the **Product details** page, a customer sees a message that the configuration is complete. By default, the message is followed by the first three descriptive attributes set in the configurator. Under the attributes, the **Show** and **Hide** buttons allow expanding and collapsing of the remaining attributes, respectively. The customer can purchase the product without opening the configurator and selecting parameters if the configuration is complete.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:97]:
```suggestion
If the configuration is complete, on the *Product details* page, a customer sees a message that the configuration is complete. By default, the message is followed by the first three descriptive attributes set in the configurator. Under the attributes, the **Show** and **Hide** buttons allow the customer to expand and collapse the remaining attributes to review them. The customer can purchase the product without again opening the configurator and selecting parameters, if they determine the configuration is complete.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 8]
```diff
@@ -89,26 +80,25 @@ Display parameter values are usually converted from configuration parameter valu
...
 
 ![configurtion-complete-message](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/Features/Configurable+Product+feature+overview/configurtion-complete-message.png)
 
-
-If the configuration is not complete, on entering the **Product details** page, a customer sees a message that the configuration is not complete. To purchase the product, they open the configurator and select a configuration. However, they can add a product with incomplete configuration to a wishlist. In this case, they can finish the configuration from the **Wishlist** page.
+If the configuration is complete, on entering the **Product details** page, a customer sees a message that the configuration needs to be completed. To purchase the product, they open the configurator and select a configuration. However, they can add a product with an incomplete configuration to a wishlist. In this case, they can finish the configuration from the **Wishlist** page.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:101]:
```suggestion
If the configuration is not complete, on the *Product details* page, a customer sees a message that the configuration needs to be completed. To purchase the product, they open the configurator and select a configuration. However, they can add a product with an incomplete configuration to a wishlist. In this scenario, they can finish the configuration from the *Wishlist* page.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 9]
```diff
@@ -117,25 +107,32 @@ Even if all the parameter values are [preconfigured](#preconfigured-parameter-va
...
 
 ### Preconfigured parameter values
 
-When a developer creates configurable products by importing them, they can pre-configure parameter values. If a customer chooses to configure such a product, they start with the preconfigured parameter values and can change them.
+When a developer creates configurable products by importing them, they can preconfigure parameter values. If customers choose to configure such a product, they start with the preconfigured parameter values and can change them.
 
-If a developer also defines that the configuration of such a product is complete, on entering the **Product details** page, a customer sees the preconfigured parameter values. They can add the product to the cart without adjusting the configuration.
+If a developer also defines that the configuration of such a product is complete, a customer sees the preconfigured parameter values on entering the **Product details** page. They can add the product to the cart without adjusting the configuration.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:126]:
```suggestion
If a developer also defines that the configuration of such a product is complete, a customer sees the preconfigured parameter values on the *Product details* page. They can add the product to the cart without adjusting the configuration.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 10]
```diff
@@ -117,25 +107,32 @@ Even if all the parameter values are [preconfigured](#preconfigured-parameter-va
...
 
-If a developer defines that the configuration of such a product is incomplete, on entering the **Product details** page, a customer does not see the preconfigured parameter values. However, they are still assigned to the product. The customer has to configure the product, but they do not have to change the preconfigured parameter values.
+If a developer defines that the configuration of such a product needs to be completed, on entering the **Product details** page, a customer does not see the preconfigured parameter values. However, they are still assigned to the product. The customer has to configure the product, but they can keep the same preconfigured parameter values.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:128]:
```suggestion
If a developer defines that the configuration of such a product needs to be completed, on the *Product details*
* page a customer does not see the preconfigured parameter values. However, they are still assigned to the product. The customer has to configure the product, but they can keep the same preconfigured parameter values.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 11]
```diff
@@ -41,20 +35,18 @@ Configurable products are created in two steps:
 1. A Back Office user creates regular products, or a developer imports them. See [Creating an abstract product](/docs/scos/user/back-office-user-guides/{{page.version}}/catalog/products/manage-abstract-products-and-product-bundles/create-abstract-products-and-product-bundles.html) to learn how they create products in the Back Office or [File details: product_concrete.csv](/docs/scos/dev/data-import/{{page.version}}/data-import-categories/catalog-setup/products/file-details-product-concrete.csv.html) to learn about the file they import.
```
Comment (michelleegan11) [inline docs/scos/user/features/202204.0/configurable-product-feature-overview.md:35]:
```suggestion
1. A Back Office user creates regular products, or a developer imports them. See [Creating an abstract product](/docs/scos/user/back-office-user-guides/{{page.version}}/catalog/products/manage-abstract-products-and-product-bundles/create-abstract-products-and-product-bundles.html) to learn how products are created in the Back Office or [File details: product_concrete.csv](/docs/scos/dev/data-import/{{page.version}}/data-import-categories/catalog-setup/products/file-details-product-concrete.csv.html) to learn about how developers import the file.
```

File: docs/scos/user/features/202204.0/configurable-product-feature-overview.md [hunk 12]
```diff
@@ -63,12 +55,11 @@ You can create a product configurator as a part of your shop or integrate a thir
 
 ![examplary-product-configurator](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Configurable+Product/Configurable+Product+feature+overview/examplary-product-configurator.png)
```
[the remark this answers is not in the document]
Comment (jeremyfourna) [reply docs/scos/user/features/202204.0/configurable-product-feature-overview.md:56]: Fine with me, please commit directly in the file, you do not need my approval for each small changes

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**